docs/perf_counters.md

```diff
@@ -12,16 +12,17 @@ This feature is available if:
 * The benchmark is run on an architecture featuring a Performance Monitoring
   Unit (PMU),
 * The benchmark is compiled with support for collecting counters. Currently,
-  this requires [libpfm](http://perfmon2.sourceforge.net/) be available at build
-  time
+  this requires [libpfm](http://perfmon2.sourceforge.net/), which is built as a
+  dependency via Bazel.
 
 The feature does not require modifying benchmark code. Counter collection is
 handled at the boundaries where timer collection is also handled. 
 
 To opt-in:
-
-*  Install `libpfm4-dev`, e.g. `apt-get install libpfm4-dev`.
-*  Enable the cmake flag BENCHMARK_ENABLE_LIBPFM.
+* If using a Bazel build, add `--define pfm=1` to your buid flags
```

Comment:
you may want to change the default to 0 if we're going to document the "opt-in" procedure. wdyt?

Reply:
If pfm=0 is specified or if no define for pfmm is provided, it will be disabled, which is the default behavior in the config_setting.

Reply:
```
config_setting(
    name = "perfcounters",
    define_values = {
        "pfm": "1",
    },
    visibility = [":__subpackages__"],
)
```

doesn't this mean it's defaulted to "1"?

Reply:
That config_setting's value to match is 1, but if you see where I use it:

```
    defines = [
        "BENCHMARK_STATIC_DEFINE",
    ] + select({
        ":perfcounters": ["HAVE_LIBPFM"],
        "//conditions:default": [],
    }),
    deps = select({
        ":perfcounters": ["@libpfm//:libpfm"],
        "//conditions:default": [],
    }),
```

We don't enable PFM or take a dependency on it if unless config_setting matches to true.

Reply:
ok! thanks